.circleci/config.yml

```diff
@@ -1,13 +1,13 @@
 version: 2
 jobs:
-  MacOS:
+  MacOS-Swift-4.2:
     macos:
       xcode: "10.1.0"
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-spm-deps-{{ checksum "Package.swift" }}
+            - macos-swift4.2-v1-spm-deps-{{ checksum "Package.swift" }}
```

Comment:
I don't think we need the "macos-" and "linux-" prefixes

Reply:
As discussed, lets keep them :)